### DISCUSSION TITLE
MAME v0.272 screen flicker issue (Intel macOS Sequoia 15.2)

### DISCUSSION THREAD

**Discussion by wyatt-wong** [mamedev/discussions#168] [General]
I built MAME from GitHub source into binary on Intel macOS Sequoia 15.2. Everytime when I execute MAME from Terminal, I got a screen flicker (screen turns total dark and then returns to my original screen, then screen display MAME GUI).

I execute mame with -v parameter and captured the log below. So why will a screen flicker happen and how to resolve it ?

https://github.com/user-attachments/files/18219378/mame.log

![Screenshot 2024-12-22 at 1 09 22 AM](https://github.com/user-attachments/assets/5532d368-8e8e-4fa2-bb3f-d04f01fdb517)

**Comment by rb6502** [comment]
Is that a hackintosh?  I didn't know Sequoia supported anything with NVIDIA graphics given Apple's hated them for like 15 years now.  The transition is seamless on my M3 Max so your flicker appears to be macOS's problem rather than MAME's.  I think I've seen that flicker in the past on beta versions of macOS but 15.2 is solid for me.

**Reply by wyatt-wong** [reply]
> Is that a hackintosh? I didn't know Sequoia supported anything with NVIDIA graphics given Apple's hated them for like 15 years now. The transition is seamless on my M3 Max so your flicker appears to be macOS's problem rather than MAME's. I think I've seen that flicker in the past on beta versions of macOS but 15.2 is solid for me.

I don't use Hackintosh. I upgraded to macOS Sequoia using OpenCore Legacy Patcher.

Furthermore, as you mention you are using M3 Max which runs on Apple Silicon, so you are NOT using the SAME MAME for Intel macOS that I manually compiled from GitHub source. You cannot compare Apple with Orange.
